pkg/reconciler/reconcile.go

```diff
@@ -9,6 +9,22 @@ import (
 	"github.com/go-logr/logr"
 )
 
+type FleetReconciler struct {
+	ctx context.Context
+	// Data required for reconcile phases
+}
+
+func (r *FleetReconciler) Reconcile() (forman.Result, error) {
```

Comment:
struct and method need to be exposed? most of the time we do name the receiver as the first letter of the struct if it makes sense to change it to `f`

Reply:
done.